config/locales/de.yml

```diff
@@ -196,6 +196,8 @@ de:
   reverse_dependencies:
     index:
       title:
+    reverse_dependencies:
```

Comment:
Yes. It is used in `reverse_dependencies/_reverse_dependencies.html.erb`. Second `reverse_dependencies` is for the partial.